app/views/patients/show.html.erb

```diff
@@ -143,9 +144,21 @@
             </div>
           </td>
         </tr>
-        <% end -%>
+          <% end %>
+        <% else %>
+          <tr>
```

Comment:
Why not just a single empty `<tr>`?

Reply:
this was another ORAU addition so also not sure

Reply:
I don't really think this new if-else is necessary at all. Doesn't appear to be doing anything. Let's make sure to follow-up on stuff like this before adding them so we understand the change.

Reply:
removed